src/libsystemd/sd-bus/bus-dump.c

```diff
@@ -81,6 +90,8 @@ _public_ int sd_bus_message_dump(sd_bus_message *m, FILE *f, uint64_t flags) {
                 if (m->reply_cookie != 0)
                         fprintf(f, "  ReplyCookie=%" PRIu64, m->reply_cookie);
 
+                fprintf(f, "  Timestamp=\"%s\"", strna(p));
```

Comment:
given this is user-facing (in contrast to the json case) I think it's correct to only show calendar time (i.e. a formatted timestamp in CLOCK_REALTIME only) here. but the json stuff above should be fixed

Reply:
change 537056f reverts the TS format in the json output to JSON_BUILD_UNSIGNED.                                                                          
I had a look into bus_message_pcap_frame() where either m->realtime or gettimeofday() is used for the timestamp.
So far I did not check the internals of m.monotonic / m.realtime and decided to use the same as in the pcap output.
Should the timestamp in pcap/text/json output be based on the same time source?